de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5479430_sys_gh3137_Drop_Processed_Column_C_PaySelectionLine_DB.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE C_PaySelectionLine DROP COLUMN Processed;
```

Comment:
please *always* use the function `public.db_alter_table(p_table_name text, p_ddl text)` when doing DDL like this, because on some DBs, this table might be partof some view